modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/FlagHighConfidenceOrthologs.pm

```diff
@@ -183,8 +183,6 @@ sub _write_distribution {
     my $n_tot = 0;
```

Comment:
It feels odd that this method is still called `_write_distribution` when it doesn't actually do that anymore

Reply:
In fact, I kinda feel like the entire method should be removed and all functionality moved to the separate `_write_distribution` methods